cmd/main.go

```diff
@@ -261,6 +271,13 @@ func startServiceController(mgr manager.Manager, nsxClient *nsx.Client) {
 	if cf.EnableAntreaNSXInterworking {
 		StartNSXServiceAccountController(mgr, commonService)
 	}
+
+	// Update pod labels to determine if this pod is the master
+	err := updatePodLabels(mgr)
+	if err != nil {
+		log.Error(err, "Failed to update pod labels")
```

Comment:
Proper nouns are better capitalized—for example, `Pod`, `Namespace`, etc.

Reply:
ok